tests/tests-core/src/main/java/com/gentics/mesh/core/field/binary/BinaryFieldEndpointTest.java

```diff
@@ -567,6 +568,20 @@ public void downloadDeletedBinary() throws IOException {
 			NOT_FOUND, "node_error_binary_data_not_found");
 	}
 
+	@Test
+    public void testTransformBinarySuccessful() throws IOException {
```

Comment:
This test has been added during the investigation of an original issue. Since I found no existing test case, covering the similar issue for the plain binaries, I left it along.